samples/tab-channel-group/mvc-csharp/README.md

```diff
@@ -29,6 +29,19 @@ In this quickstart we'll walk-through creating a custom channel/group tab with A
   
 - [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
 
+- Teams Toolkit
```

Comment:
`Teams Toolkit for Visual Studio`, and also better to add a link.

Reply:
Fixed